pkg/utils/activity/config.go

```diff
@@ -0,0 +1,33 @@
+package activity
+
+import (
+	"time"
+
+	"github.com/trisacrypto/directory/pkg/utils/ensign"
+)
+
+type Config struct {
+	Enabled           bool          `split_words:"true" default:"false"`
```

Comment:
Ok, so because this defaults to false, we don't need to modify the environment varibles in docker-compose; still it might be nice to explicitly set the docker compse ACTIVITY_ENABLED to false for future us to more easily figure out what is going on.